.github/dependabot.yml

```
@@ -9,3 +9,11 @@ updates:
       - 'pr: dependencies'
     versioning-strategy: increase
     target-branch: 'v14' # TODO: Remove when releasing v14.
+
+  - package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: weekly
+    open-pull-requests-limit: 5
+    labels:
+      - 'pr: dependencies'
```

IMO, setting `target-branch: v14` seems better. Because `v14` is currently active and there are many changes between `v14` and `master`.

```suggestion
      - 'pr: dependencies'
    target-branch: 'v14' # TODO: Remove when releasing v14.
```